migrations/20250904133000_add_oauth_client_id_to_session.up.sql

```diff
@@ -0,0 +1,2 @@
+alter table if exists {{ index .Options "Namespace" }}.sessions
+  add column if not exists oauth_client_id uuid null references {{ index .Options "Namespace" }}.oauth_clients(id) on delete cascade;
```

Comment:
This needs an index?

Reply:
I remember recently a customers own migration which touched auth tables caused an outage for them due to the table locks. If we add an index maybe we could break this down into smaller parts, maybe something like below?

```sql
alter table if exists {{ index .Options "Namespace" }}.sessions
  add column if not exists oauth_client_id uuid;

alter table {{ index .Options "Namespace" }}.sessions
  add constraint sessions_oauth_client_id_fkey foreign key (oauth_client_id)
  references {{ index .Options "Namespace" }}.oauth_clients(id) on delete cascade not valid;

alter table {{ index .Options "Namespace" }}.sessions
  validate constraint sessions_oauth_client_id_fkey;
 ```